apps/web/src/app/admin-console/organizations/policies/policies.component.ts

```diff
@@ -37,27 +40,23 @@ export class PoliciesComponent implements OnInit {
 
   private orgPolicies: PolicyResponse[];
   protected policiesEnabledMap: Map<PolicyType, boolean> = new Map<PolicyType, boolean>();
+  protected isUpsellingEnabled: boolean;
 
   constructor(
     private route: ActivatedRoute,
     private organizationService: OrganizationService,
-    private accountService: AccountService,
     private policyApiService: PolicyApiServiceAbstraction,
     private policyListService: PolicyListService,
+    private organizationBillingService: OrganizationBillingServiceAbstraction,
     private dialogService: DialogService,
   ) {}
 
   async ngOnInit() {
     // eslint-disable-next-line rxjs-angular/prefer-takeuntil, rxjs/no-async-subscribe
     this.route.parent.parent.params.subscribe(async (params) => {
       this.organizationId = params.organizationId;
-      const userId = await firstValueFrom(
-        this.accountService.activeAccount$.pipe(map((a) => a?.id)),
-      );
       this.organization = await firstValueFrom(
-        this.organizationService
-          .organizations$(userId)
-          .pipe(getOrganizationById(this.organizationId)),
+        this.organizationService.getOrganizationById$(this.organizationId),
```

Comment:
We should fix this to use the existing function in the organizaiton service:
```suggestion
this.organization$ = this.accountService.activeAccount$.pipe(
        getUserId,
        switchMap(userId => this.organizationService.organizations$(userId)),
        getOrganizationById(this.organizationId)
      );
```